app/server.js

```diff
@@ -2,6 +2,8 @@ const app = require('express')();
 const nconf = require('nconf');
 const globSync = require('glob').sync;
 
+const logger = require('./services/logger');
```

Comment:
are you sure that logger is a service?

Reply:
I moved logger to `utils` folder. See [commit](https://github.com/inst-event-emitter/eventemitter-backend/pull/7/commits/e02b618176eba1d6fdcc919149f268e3cea63629#diff-f53e59f241bb62f0261e245350683e07R5)